cmd/ssl-expiry-check/main.go

```diff
@@ -50,37 +87,103 @@ func init() {
 		log.Error("ERROR: The INSECURE environment variable has not been set.")
 		return
 	}
+
+	insecureBool, _ = strconv.ParseBool(insecureCheck)
+
+	// set debug mode for nodeCheck pkg
+	nodeCheck.EnableDebugOutput()
 }
 
 func main() {
-	err := runExpiry()
+	// create context
+	checkTimeLimit := time.Minute * 1
+	ctx, _ := context.WithTimeout(context.Background(), checkTimeLimit)
```

Comment:
so this ctx is for the nodeCheck -- maybe we can rename it to that? since its confusing with the ctx you reset below for the actual check run.

also maybe rename `checkTimeLimit` to `nodeCheckTimeout`?

Reply:
Updated, PR resubmitted.